tests/integ/modin/binary/test_binary_op.py

```diff
@@ -1299,10 +1298,13 @@ def check_op(native_lhs, native_rhs, snow_lhs, snow_rhs):
             snow_ans, native_ans, lambda s: s, check_index_type=False
         )
 
-    check_op(lhs, rhs, try_cast_to_snow_series(lhs), try_cast_to_snow_series(rhs))
+    with SqlCounter(
+        query_count=2, join_count=10 if isinstance(lhs.index, pd.MultiIndex) else 6
```

Comment:
why is the join count so high?

Reply:
The single index series are created with one join each - these series are referred to in the query with their joins. So 2 joins performed for creating 2 series (happens twice), 1 join per binary op.

With MultiIndex, the number of joins performed is equal to the number of columns in the MultiIndex. So two joins are performed per series created. Two series created twice, which is 4*2, 8 joins. One joins for each binary operation (two binary operations performed)